README.md

```diff
@@ -101,14 +132,102 @@ credentials, ie:
 }
 ```
 
-## Publishing
+The contents of this file can be found in lastpass with the name
```

Comment:
Should this entry ever be lost, it might be helpful to describe the process necessary to regenerate / refetch these credentials.